README.md

```diff
@@ -1,2 +1,3 @@
 # Dim
 try project
+edited this line to learn the process
```

Comment:
line added for practice, No issue. 
can be merged.

Reply:
should I merge >>?